### ISSUE TITLE
Archival Storage – bulk input AIP to Archival Storage from Ingest using script

### ISSUE THREAD

**Issue by audreyhzhang** [gbhl/bhl-europe#21]
The Ingest component creates AIP's and stores them in a working folder for automated input into fedora via a script.

**Comment by leenamba** [comment]
will wait for Bernd to indicate where AIP's are on NFS share
@sprogerb

**Comment by janahoffmann** [comment]
Hengdi, Lee can I remove the label feature from this issue as this might be confusing. I am not sure what this is exactly about (decription?). Maybe you can provide a description....

**Comment by leenamba** [comment]
I added a description.  We can remove the feature label but I would consider being able to preserve objects in BHL Europe a core feature.

**Comment by janahoffmann** [comment]
I totally agree. We just have to distinguish between portal features and other 'features' related to the overall BHL-Europe system. So, I remove the label, but will leave it untouched...:-)

**Comment by sprogerb** [comment]
@leenamba Indicated and communicated to Atos where and how to retrieve AIP METS files

**Comment by leenamba** [comment]
Will need further modification now that pre-ingest is being refactored.  So this issue will most likely stay open post november milestone.
